lib/openscap_parser/profiles.rb

```diff
@@ -1,25 +1,21 @@
 # frozen_string_literal: true
-require 'openscap_parser/test_result'
 
 module OpenscapParser
   # Methods related to saving profiles and finding which hosts
   # they belong to
   module Profiles
-    include TestResult
-
     def self.included(base)
       base.class_eval do
         def profiles
-          @profiles ||= {
-            profile_node['id'] => profile_node.at_css('title').text
```

Comment:
This method is now `test_result_profiles` from the `TestResult` module. It returns the list of (one) profiles that this scan was run against.